developer/src/kmc-convert/src/converter-messages.ts

```diff
@@ -23,31 +23,26 @@ export class ConverterMessages {
 
   static ERROR_NoConverterFound = SevError | 0x0002;
   static Error_NoConverterFound = (o: { inputFilename: string, outputFilename: string; }) => m(
-    this.ERROR_NoConverterFound,
-    `No converter is available that can convert from '${def(o.inputFilename)}' to '${def(o.outputFilename)}'.`);
+    this.ERROR_NoConverterFound, `No converter is available that can convert from '${def(o.inputFilename)}' to '${def(o.outputFilename)}'.`);
```

Comment:
This is the layout we use for the *-messages.ts files now (here and below):

```suggestion
    this.ERROR_NoConverterFound, 
    `No converter is available that can convert from '${def(o.inputFilename)}' to '${def(o.outputFilename)}'.`,
  );
```